turbo.json

```diff
@@ -6,10 +6,6 @@
       "outputs": ["build/**", "dist/**"],
       "dependsOn": ["^build"]
     },
-    "test": {
-      "outputs": ["coverage/**"],
-      "dependsOn": []
-    },
```

Comment:
We can remove empty `dependsOn`.

We don't run `test` task to produce coverage. If we want to support running test with coverage artifacts, we should create a separate `test:coverage` task that outputs to `"coverage/**"`.